include/fe/fe.h

```diff
@@ -686,7 +690,9 @@ class FEHierarchic : public FE<Dim,HIERARCHIC>
    * to be used in dimension \p Dim.
    */
   explicit
-  FEHierarchic(const FEType& fet);
+  FEHierarchic(const FEType& fet) :
+    FE<Dim,HIERARCHIC> (fet)
+  {}
```

Comment:
Yeah, these changes should be in a separate patch, so they can be reverted if you want, but I highly prefer this version.  It's easier to read and about 75% less lines for one-liner constructors.